packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -83,6 +84,12 @@ class FlatTable<T> extends StatefulWidget {
 
   final List<ColumnGroup>? columnGroups;
 
+  /// Determines if the headers for column groups should be rendered.
+  ///
+  /// If set to false and `columnGroups` is non-null and non-empty, only
+  /// dividing lines will be drawn for each column group boundary.
```

Comment:
super nit: can you change to "only vertical dividing lines" to make it clear which divider you are referring to. When I first read this I thought we were talking about the horizontal divider lines between column group headers and column headers.